updater/osv/osv.go

```diff
@@ -484,6 +484,12 @@ func newECS(u string) ecs {
 	}
 }
 
+type rangeVer struct {
+	fixedInVersion  string
+	semverRange     *claircore.Range
+	ecosystemRanges url.Values
```

Comment:
nit: should they both be singular/plural, or does this make most sense as-is?

Reply:
Both singular I think :+1: